openacademy/models/models.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
```

Comment:
Please avoid import things that you don't use, like `api`. 

I know you'll use it later, but don't import it for now.